.travis.yml

```diff
@@ -51,15 +51,16 @@ matrix:
 before_install:
   - "travis_retry pip install --upgrade pip setuptools py"
   - "travis_retry pip install twine wheel coveralls requirements-builder"
-  - "requirements-builder --level=min setup.py > .travis-lowest-requirements.txt"
-  - "requirements-builder --level=pypi setup.py > .travis-release-requirements.txt"
-  - "requirements-builder --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
+  - "requirements-builder -e ${EXTRAS} --level=min setup.py > .travis-lowest-requirements.txt"
+  - "requirements-builder -e ${EXTRAS} --level=pypi setup.py > .travis-release-requirements.txt"
+  - "requirements-builder -e ${EXTRAS} --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
 
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[${EXTRAS}]"
 
 script:
+  - pip freeze
```

Comment:
Was this intended?

Reply:
Yeah, it generally helps with resolving local vs travis dependencies when looking at travis logs